src/main/java/seedu/lifeasier/parser/Parser.java

```diff
@@ -512,14 +512,46 @@ private Command parseEditNotesCommand(String input) {
      * @param input String containing the user's input.
      * @return DisplayScheduleCommand with the parameters input by the user.
      */
-    private Command parseDisplayScheduleCommand(String input) {
-
+    private Command parseDisplayScheduleCommand(String input) throws ParserException {
+        Ui ui = new Ui();
```

Comment:
Can we find a way to do it without instantiating another Ui object? I think that when we call parseCommand from LifEasier, we already pass in a Ui object.